### PR TITLE
Noto Serif Old Uyghur: Version 1.002; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by simoncozens**
871972c: [gftools-packager] Noto Serif Old Uyghur: Version 1.002; ttfautohint (v1.8.4.7-5d5b) added

* Noto Serif Old Uyghur Version 1.002; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/old-uyghur.git at commit https://github.com/notofonts/old-uyghur/commit/.

### CODE REVIEW

File: ofl/notoserifolduyghur/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/old-uyghur)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/old-uyghur.git"
+  archive_url: "https://github.com/notofonts/old-uyghur/releases/download/NotoSerifOldUyghur-v1.002/NotoSerifOldUyghur-v1.002.zip"
+}
+is_noto: true
```
Comment (RosaWagner) [inline ofl/notoserifolduyghur/METADATA.pb:22]: @simoncozens missing subset, languages and primary script.

Comment (simoncozens) [reply ofl/notoserifolduyghur/METADATA.pb:22]: That suggests we have no lang data for this script. I'll see what I can do.

**Comment by RosaWagner** [comment]
@simoncozens do we have now language data for Old Uyghur?

**Comment by simoncozens** [comment]
No, and it's going to be tricky. :-( I could try Andrew West, or we could try transliterating from another script.

**Comment by simoncozens** [comment]
Updated

## Noto Serif Old Uyghur: Version 1.003; ttfautohint (v1.8.4.7-5d5b) added

---
 a1bf7e2: [gftools-packager] Noto Serif Old Uyghur: Version 1.003; ttfautohint (v1.8.4.7-5d5b) added

* Noto Serif Old Uyghur Version 1.003; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/old-uyghur.git at commit https://github.com/notofonts/old-uyghur/commit/.

File: ofl/notoserifolduyghur/METADATA.pb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "old-uyghur"
+source {
+  repository_url: "https://github.com/notofonts/old-uyghur.git"
+  archive_url: "https://github.com/notofonts/old-uyghur/releases/download/NotoSerifOldUyghur-v1.003/NotoSerifOldUyghur-v1.003.zip"
+}
+is_noto: true
+primary_script: "Ougr"
```
Comment (RosaWagner) [inline ofl/notoserifolduyghur/METADATA.pb:24]: @simoncozens are there now language data to add here?

Comment (simoncozens) [reply ofl/notoserifolduyghur/METADATA.pb:24]: [Should be](https://github.com/googlefonts/lang/blob/main/Lib/gflanguages/data/languages/oui_Ougr.textproto).

**Pull request merged**